flappy_bird_example/metal_owl.py

```diff
@@ -0,0 +1,227 @@
+# import all these things so that the .pv file can access them
+
+import random # import random allows for random generation of values
+
+from kivy.app import App # import abilities of kivy to run app
+from kivy.uix.widget import Widget # import widgets from kivy
+#from kivy.graphics import Color, Rectangle # commented out as image used instead
+from kivy.core.window import Window # import window adjustment
+from kivy.uix.image import Image # import use of images
+from kivy.clock import Clock # import kivy clock
+from kivy.uix.label import Label # import kivy label
+from kivy.core.audio import SoundLoader # import sound usage
+
+# set the sound source
+sfx_flap = SoundLoader.load("Resources/kivy-game-dev/flappy/audio/flap.wav")
+sfx_score = SoundLoader.load("Resources/kivy-game-dev/flappy/audio/score.wav")
+sfx_die = SoundLoader.load("Resources/kivy-game-dev/flappy/audio/die.wav")
+
+# create the main menu using the widget to add the background, ground, label
+class Menu(Widget):
+    def __init__(self):
+        super(Menu, self).__init__()
+        self.add_widget(Sprite(source="Resources/kivy-game-dev/flappy/images/background.png"))
+        self.size = self.children[0].size
```

Comment:
If the you used kv language to give the background an id and made it a child of this widget, you could refer to it via its id instead. children[0] makes it hard to know what its referring to without reading the code, and could be an issue for maintainability if a widget had many child widgets.